docs/quickstart.rst

```diff
@@ -95,6 +95,44 @@ borg.
 If you need to access a local repository from different users, you can use the
 same method by using ssh to borg@localhost.
 
+Important note about files changing during the backup process
+-------------------------------------------------------------
+
+Borg does not do anything about the internal consistency of the data
+it backs up.  It just reads and backs up each file in whatever state
+that file is when Borg gets to it.  On an active system, this means
+that by the time Borg backs up a file, it might have changed since the
+backup process was initiated.
```

Comment:
This is only one of the problems (1 file being not consistent with other files).

Even more severe is usually if 1 file is not internally consistent because it has changed while being backed up.